### PR TITLE
LXDE small modifications

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/lxde/core/lxrandr/default.nix [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+    libX11
+    xrandr
+    (if withGtk3 then gtk3 else gtk2)
+  ];
 
   meta = with lib; {
     description = "Standard screen manager of LXDE";
     homepage = "https://lxde.org/";
-    license = lib.licenses.gpl2;
+    license = licenses.gpl2;
```
Comment (SuperSandro2000) [inline pkgs/desktops/lxde/core/lxrandr/default.nix:33]: gpl2Only or gpl2Plus?

File: pkgs/development/libraries/libfm/default.nix [hunk 2]
```diff
@@ -8,8 +8,9 @@
 , pkg-config
 , vala
 , extraOnly ? false
-, withGtk3 ? true , gtk3, gtk2
+, withGtk3 ? false, gtk2, gtk3
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libfm/default.nix:11]: You are sure about this change?

File: pkgs/desktops/lxde/core/lxrandr/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/desktops/lxde/core/lxrandr/default.nix:33]: Oh, I almost forget.

File: pkgs/development/libraries/libfm/default.nix (hunk 2 above)
Comment (AndersonTorres) [reply pkgs/development/libraries/libfm/default.nix:11]: It triggers other dependencies, and they call both libfm and gtk. If they aren't in sync, the build fails.

An example is lxpanel. It doesn't build with libfm-gtk3.

Comment (AndersonTorres) [reply pkgs/development/libraries/libfm/default.nix:11]: (Maybe LXDE needs some "global" variable to handle this.)

**Pull request merged**